cli/pause.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+// pauseDescription is used to describe pause command in detail and auto generate command doc.
+var pauseDescription = "Pause a running container object in Pouchd. " +
+	"when pausing, the container will pause its running but hold all the relevant resource." +
+	"This is useful when you wish to pause a container for a while and to restore the running status later." +
+	"The container you paused will pause without being terminated."
+
+// PauseCommand use to implement 'pause' command, it pauses a container.
+type PauseCommand struct {
+	baseCommand
+}
+
+// Init initialize pause command.
+func (p *PauseCommand) Init(c *Cli) {
+	p.cli = c
+	p.cmd = &cobra.Command{
+		Use:   "pause [container]",
+		Short: "Pause a running container",
+		Long:  pauseDescription,
+		Args:  cobra.MinimumNArgs(1),
```

Comment:
I think your command takes exact one argument. Then here need to be `cobra.ExactArg(1)`, rather than `cobra.MinimumNArgs(1)`.

Reply:
I tried the pause feature on docker, it seems that docker can pause several images at the same time. In other words, more than one argument can be taken within one command line there. So I went with cobra.MinimumNArgs(1).

Reply:
I have tried the pause feature on docker, and it seems that docker can pause several images at the same time. In other words, more than one argument can be taken within one command line there. So I went with cobra.MinimumNArgs(1).

Reply:
Docker could pause multiple containers. It is only on cli side, not API side.
In docker, code takes more than one arguments and deal all of them. But in your code here, you just pass one argrment in 
```
func (p *PauseCommand) runPause(args []string) error {
     apiClient := p.cli.Client()
 
     container := args[0]
```

Exact one and multiple ones are both right. However, I am afraid we should keep consistent. WDYT?

Reply:
I have tried the pause feature on docker, and it seems that docker can pause several containers at the same time. In other words, more than one argument can be taken within one command line there. So, I went with cobra.MinimumNArgs(1).

Reply:
Yeah. I see...